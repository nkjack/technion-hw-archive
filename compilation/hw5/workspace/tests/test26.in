//func args
int foo(int w, byte x, bool c, int y) {
	printi(w);
	print("\t");
	printi(x);
	print("\t");
	if (c) print("true\t"); else print("false\t");
	printi(y);
	print("\n");
	return 0;
}

void main() {
	foo(4,7b,true,15);
	foo(1b,6b,false,15);
	byte x = 89b;
	byte y = 0b;
	bool z = true;
	foo(x,y,z,x);
	foo(x + 25,y - 53b,88 >= x, 125 * x * (y+12));
}