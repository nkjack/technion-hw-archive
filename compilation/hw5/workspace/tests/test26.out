Loaded: ./exceptions.s
4	7	true	15
1	6	false	15
89	0	true	89
114	203	false	133500
